src/ol/uri.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('ol.uri');
+
+
+/**
+ * Appends query parameters from a map to an URI.
```

Comment:
I would avoid the use of the term "map" here.  In the case of our library, this is too easily confused with an instance of `ol.Map`.  In terms of the language, this is too easily confused with an instance of `Map`.  Instead, this could read:

> Appends query parameters to a URL.